setup.py

```diff
@@ -195,6 +195,9 @@ def find_any_distribution(pkgs: List[str]) -> Optional[pkg_resources.Distributio
     author="Takuya Akiba",
     author_email="akiba@preferred.jp",
     url="https://optuna.org/",
+    project_urls={
+        "Source": "https://github.com/optuna/optuna",
```

Comment:
This will show like this:

<img width="363" alt="Screenshot 2022-03-02 at 23 42 02" src="https://user-images.githubusercontent.com/7121753/156383798-90ef8739-ed02-4184-bd5c-8b18f92f16ab.png">

From https://pypi.org/project/matplotlib/
Source: https://github.com/matplotlib/matplotlib/blob/a8534072ccf123c2651336b003baa1e4ce47a481/setup.py#L267-L273